### PR TITLE
Mood HUD changes color depending on sanity

### PR DESCRIPTION
**Pull request by Mickyan**
##  About The Pull Request

Removes the sanity outline and makes it so the mood icon changes color based on sanity, while the shape/expression changes based on mood

## Why It's Good For The Game
Looks better and makes the more important stat (sanity) more prevalent on the hud at a glance

It also means special mood icons don't necessarily need to conform to the standard face shape without clipping into the sanity icon (hint hint)

So basically good mood and low sanity will look like this
Screenshot_3

## Changelog
:cl:
tweak: The way mood and sanity is displayed on the HUD has been changed
tweak: Sanity level will now be displayed by color changes, while facial expression displays current mood
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
epic pr
